From e2c353d11190579bd0ffd5dee3ba2a3b707a0d92 Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Sun, 23 Oct 2016 09:25:55 +0300
Subject: [PATCH 18/23] ld: testsuite: Relocatable output for PRU actually
 merges all .text.* sections

Note to self: Check how AVR port's usage of different linker scripts.
For example, "vimdiff avr5.x avr5.xr"

2016-10-23  Dimitar Dimitrov <dimitar@dinux.eu>

	* ld/testsuite/ld-elf/pr20528a.d: Disable for PRU.
	* ld/testsuite/ld-elf/pr20528b.d: Ditto.

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 ld/testsuite/ld-elf/pr20528a.d | 2 +-
 ld/testsuite/ld-elf/pr20528b.d | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ld/testsuite/ld-elf/pr20528a.d b/ld/testsuite/ld-elf/pr20528a.d
index 8ffb816..b5dc4c2 100644
--- a/ld/testsuite/ld-elf/pr20528a.d
+++ b/ld/testsuite/ld-elf/pr20528a.d
@@ -3,7 +3,7 @@
 #ld: -r
 #readelf: -S --wide
 #xfail: d30v-*-* dlx-*-* fr30-*-* frv-*-elf ft32-*-* i860-*-* i960-*-*
-#xfail: iq*-*-* mn10200-*-* moxie-*-* msp*-*-* mt-*-* pj*-*-*
+#xfail: iq*-*-* mn10200-*-* moxie-*-* msp*-*-* mt-*-* pj*-*-* pru-*-*
 
 #...
 [ 	]*\[.*\][ 	]+\.text.startup[ 	]+PROGBITS.*[ 	]+AX[   ]+.*
diff --git a/ld/testsuite/ld-elf/pr20528b.d b/ld/testsuite/ld-elf/pr20528b.d
index 80f0aa5..c0f7c25 100644
--- a/ld/testsuite/ld-elf/pr20528b.d
+++ b/ld/testsuite/ld-elf/pr20528b.d
@@ -3,7 +3,7 @@
 #ld: -r
 #readelf: -S --wide
 #xfail: d30v-*-* dlx-*-* fr30-*-* frv-*-elf ft32-*-* i860-*-* i960-*-*
-#xfail: iq*-*-* mn10200-*-* moxie-*-* msp*-*-* mt-*-* pj*-*-*
+#xfail: iq*-*-* mn10200-*-* moxie-*-* msp*-*-* mt-*-* pj*-*-* pru-*-*
 
 #...
 [ 	]*\[.*\][ 	]+\.text.startup[ 	]+PROGBITS.*[ 	]+AXE[   ]+.*
-- 
2.10.2

